src/overlays/actors/ovl_En_Ny/z_en_ny.c

```diff
@@ -50,53 +61,542 @@ static ColliderJntSphInit D_80ABE354 = {
+static DamageTable sDamageTable = {
+    /* Deku nut      */ DMG_ENTRY(0 , 0x0),
+    /* Deku stick    */ DMG_ENTRY(0 , 0x0),
+    /* Slingshot     */ DMG_ENTRY(0 , 0x0),
+    /* Explosive     */ DMG_ENTRY(2 , 0xF),
+    /* Boomerang     */ DMG_ENTRY(0 , 0x0),
+    /* Normal arrow  */ DMG_ENTRY(2 , 0xF),
+    /* Hammer swing  */ DMG_ENTRY(2 , 0xF),
+    /* Hookshot      */ DMG_ENTRY(2 , 0x1),
+    /* Kokiri sword  */ DMG_ENTRY(0 , 0x0),
+    /* Master sword  */ DMG_ENTRY(2 , 0xF),
+    /* Giant's Knife */ DMG_ENTRY(4 , 0xF),
+    /* Fire arrow    */ DMG_ENTRY(4 , 0x2),
+    /* Ice arrow     */ DMG_ENTRY(2 , 0xF),
+    /* Light arrow   */ DMG_ENTRY(2 , 0xF),
+    /* Unk arrow 1   */ DMG_ENTRY(4 , 0xE),
+    /* Unk arrow 2   */ DMG_ENTRY(0 , 0x0),
+    /* Unk arrow 3   */ DMG_ENTRY(0 , 0x0),
+    /* Fire magic    */ DMG_ENTRY(4 , 0x2),
+    /* Ice magic     */ DMG_ENTRY(0 , 0x0),
+    /* Light magic   */ DMG_ENTRY(0 , 0x0),
+    /* Shield        */ DMG_ENTRY(0 , 0x0),
+    /* Mirror Ray    */ DMG_ENTRY(0 , 0x0),
+    /* Kokiri spin   */ DMG_ENTRY(0 , 0x0),
+    /* Giant spin    */ DMG_ENTRY(4 , 0xF),
+    /* Master spin   */ DMG_ENTRY(2 , 0xF),
+    /* Kokiri jump   */ DMG_ENTRY(0 , 0x0),
+    /* Giant jump    */ DMG_ENTRY(8 , 0xF),
+    /* Master jump   */ DMG_ENTRY(4 , 0xF),
+    /* Unknown 1     */ DMG_ENTRY(0 , 0x0),
+    /* Unblockable   */ DMG_ENTRY(0 , 0x0),
+    /* Hammer jump   */ DMG_ENTRY(0 , 0x0),
+    /* Unknown 2     */ DMG_ENTRY(0 , 0x0),
+};
+
+static InitChainEntry sInitChain[] = {
+    ICHAIN_S8(naviEnemyId, 40, ICHAIN_CONTINUE),
+    ICHAIN_U8(targetMode, 2, ICHAIN_CONTINUE),
+    ICHAIN_F32(targetArrowOffset, 30, ICHAIN_STOP),
+};
+
+void EnNy_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnNy* this = THIS;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+    this->actor.colChkInfo.damageTable = &sDamageTable;
+    this->actor.colChkInfo.health = 2;
+    Collider_InitJntSph(globalCtx, &this->collider);
+    Collider_SetJntSph(globalCtx, &this->collider, &this->actor, &sColliderInit, &this->elements);
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 20.0f);
+    this->unk_1CA = 0;
+    this->unk_1D0 = 0;
+    Actor_SetScale(&this->actor, 0.01f);
+    this->actor.speedXZ = 0.0f;
+    this->actor.shape.rot.y = 0;
+    this->actor.gravity = -0.4f;
+    this->hitPlayer = 0;
+    this->unk_1CE = 2;
+    this->actor.velocity.y = 0.0f;
+    this->unk_1D4 = 0xFF;
+    this->unk_1D8 = 0;
+    this->unk_1E8 = 0.0f;
+    this->unk_1E0 = 0.25f;
+    if (this->actor.params == 0) {
+        // New initials
+        osSyncPrintf("ニュウ イニシャル[ %d ] ！！\n", this->actor.params);
+        this->actor.colChkInfo.mass = 0;
+        this->unk_1D4 = 0;
+        this->unk_1D8 = 0xFF;
+        this->unk_1E0 = 1.0f;
+        func_80ABCDBC(this);
+    } else {
+        //@Bug this block is never reached because params is never set to anything other than 0
+        // Dummy new initials
```

Comment:
I wouldnt classify this as a bug, its just an unused param

Reply:
a comment isnt even worth imo, but if you want to say something you can say "this mode is unused in the final game" or something